### PR TITLE
embulk: add note for 0.10.x release

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/embulk.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Embulk < Formula
   desc "Data transfer between various databases, file formats and services"
   homepage "https://www.embulk.org/"
-  url "https://bintray.com/artifact/download/embulk/maven/embulk-0.9.23.jar"
-  sha256 "153977fad482bf52100dd96f47e897c87b48de4fb13bccd6b3101475d3a5ebb9"
+  url "https://jcenter.bintray.com/org/embulk/embulk-core/0.10.1/embulk-core-0.10.1.jar"
```
Comment (chenrui333) [inline Formula/embulk.rb:4]: This does not exist in bintry (searched in here, https://bintray.com/beta/#/embulk/maven/embulk?tab=files).

**Comment by chenrui333** [comment]
```
[34m==>[0m [1mbrew test embulk --verbose[0m
Testing embulk
/usr/bin/sandbox-exec -f /private/tmp/homebrew20200412-19003-8sn5gy.sb ruby -W0 -I $LOAD_PATH -- /usr/local/Homebrew/Library/Homebrew/test.rb /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/embulk.rb --verbose
[34m==>[0m [1m/usr/local/Cellar/embulk/0.10.1/bin/embulk example ./try1[0m
/usr/local/Cellar/embulk/0.10.1/libexec/embulk.jar: line 1: PK: command not found
/usr/local/Cellar/embulk/0.10.1/libexec/embulk.jar: line 2: ï¿½{P: command not found
/usr/local/Cellar/embulk/0.10.1/libexec/embulk.jar: line 3: ï¿½{Plï¿½ï¿½ï¿½eï¿½META-INF/MANIFEST.MFï¿½Mï¿½ï¿½LK-.ï¿½
K-*ï¿½ï¿½Ï³R0ï¿½3ï¿½ï¿½ï¿½ï¿½-ï¿½Iï¿½Mï¿½+I,: No such file or directory
/usr/local/Cellar/embulk/0.10.1/libexec/embulk.jar: line 4: syntax error near unexpected token `)'
/usr/local/Cellar/embulk/0.10.1/libexec/embulk.jar: line 4: `ï¿½dï¿½ï¿½Z)ï¿½ï¿½&ï¿½ï¿½dï¿½&ï¿½ï¿½b(ï¿½ï¿½7ï¿½3",ï¿½y)ï¿½Eï¿½ï¿½)V'
[31mError:[0m embulk: fai
```

**Comment by Bo98** [comment]
> `v0.10.*` is a “development” series, not for your production use.
> In your production, keep using `v0.9.*`.

https://www.embulk.org/articles/2020/03/13/embulk-v0.10.html

**Comment by SMillerDev** [comment]
> > `v0.10.*` is a “development” series, not for your production use.
> > In your production, keep using `v0.9.*`.
> 
> https://www.embulk.org/articles/2020/03/13/embulk-v0.10.html

Could you make a comment with that?

**Comment by chenrui333** [comment]
I see, I will pivot this PR to address the version intention.

**Comment by SMillerDev** [comment]
Might be needed to mark this in livecheck as well.

**Comment by chenrui333** [comment]
@SMillerDev great call! Created an issue and update it on the livecheck side as well.

**Pull request merged**